images/mediatypes.go

```diff
@@ -161,3 +200,12 @@ func ChildGCLabelsFilterLayers(desc ocispec.Descriptor) []string {
 	}
 	return ChildGCLabels(desc)
 }
+
+// ClearGCLabels clears GC labels for the given digest.
```

Comment:
Can you move this into the converter package? I don't think it makes sense here

Reply:
moved